scripts/fires/wfigs_data.py

```diff
@@ -295,14 +300,20 @@ def main(_) -> None:
     pre_2022_df = get_data(PRE_2022_FIRE_LOCATIONS_URL)
     df = pre_2022_df
 
-    with open(os.path.join(_SCRIPT_PATH, 'location_file.json')) as f:
-        data = f.read()
-    _LAT_LNG_CACHE = json.loads(data)
+    storage_client = storage.Client(_GCS_PROJECT_ID)
+    bucket = storage_client.bucket(_GCS_BUCKET)
+    blob = bucket.blob(_GCS_FILE_PATH)
+    if FLAGS.read_location_cache:
+        try:
+            _LAT_LNG_CACHE = json.loads(blob.download_as_string(client=None))
+        except Exception as e:  # pylint: disable=broad-except
+            logging.info("Reading locations cache failed: %e" % e)
```

Comment:
error?

Reply:
The script would still work even if we can't read the locations cache, it would just be slower. So maybe exception here is ok?

Reply:
sorry, what I mean is s/logging.info/logging.error/